### DISCUSSION TITLE
Explorer- Search file path question, why always only in 3D objects folder?

### DISCUSSION THREAD

**Discussion by Jellyfishee** [Flow-Launcher/Flow.Launcher#1137] [Q&A] [answered]
Hi everyone, I am new to Flow Launcher
Would like to see how to change the default search path to "My computer" or Desktop, instead of now using 3D object file as the default. All search comes from there and i miss a lot files that are not inside this file folder.
![image](https://user-images.githubusercontent.com/104116805/164363456-311ff733-7911-40b8-9c61-18c7a9b1d314.png)

**Comment by jjw24** [comment, marked as answer]
Hey welcome. 

This is likely due to the windows index only indexing that path. You can change this by editing the indexed path via 'Indexing Options'.

**Reply by Jellyfishee** [reply]
> Hey welcome.
> 
> This is likely due to the windows index only indexing that path. You can change this by editing the indexed path via 'Indexing Options'.

Where could i find "indexing options"? I couldn't find it in the setting. Could you show me?? How can i set up the indexed path to " all files in my computer" instead of specific path (like 3D objects folder)

**Reply by jjw24** [reply]
Sure thing, you can access indexing options via typing `indexing options` from flow. Follow this YouTube video to configure it https://youtu.be/8j-JkkZDtfw
